pkg/maps/ctmap/gc/gc.go

```diff
@@ -111,9 +112,11 @@ func Enable(ipv4, ipv6 bool, restoredEndpoints []*endpoint.Endpoint, mgr Endpoin
 				signal.MuteChannel(signal.SignalWakeGC)
 			}
 
+			triggeredBySignal = false
 			signal.UnmuteChannel(signal.SignalWakeGC)
 			select {
 			case x := <-wakeup:
+				triggeredBySignal = true
```

Comment:
Why not also purge the nat table at some regular interval in the 'case <-time.After(...)' below. It seems running from below would also help cleanup any orphaned entries before the problem is hit. Or are we trying to save CPU here?

Reply:
Answered here to keep the discussion in the single PR page: https://github.com/cilium/cilium/pull/13912/files#r519410542